src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -0,0 +1,34 @@
+package edu.kis.vh.nursery;
+
+public class DefaultCountingOutRhymer {
+
+	private int[] numbers = new int[12];
```

Comment:
Missing: Temat: Stałe i annotacje!

Reply:
Had a talk with my collaborator and added 1.4